pages/publishing/publishing/adobe-experience-platform.mdx

```diff
@@ -0,0 +1,81 @@
+import {Callout} from 'nextra/components'
+
+# Adobe Experience Platform
+
+*Publishing your tracking plan to the Adobe Experience Platform*
+
+![Adobe Logo](/images/publishing_adobe.svg)
+
+## Introduction
+
+The integration to [Adobe Experience Platform XDM system](https://experienceleague.adobe.com/en/docs/experience-platform/xdm/home) allows you to publish your Avo Tracking Plan as XDM Schemas to Adobe Experience Platform in form of XDM schemas. You can set up the integration to automatically publish on every Avo branch merge or manually trigger a publish. 
+
+## Configuring the Adobe Experience Platform integration
+
+To enable the direct integration with AEP, navigate to the Publishing Screen in your Avo workspace, click “Add Integration” and select “Adobe Experience Platform. 
+
+You will need to provide the following parameters from Adobe:
+
+- **Adobe Client Id**
+- **Adobe Organization ID**
+- **Adobe Client Secret**
+
+ These parameters can be found from the Adobe Developer Console -> Projects -> Workspaces -> Credentials. The URL should look like this: `https://developer.adobe.com/console/projects/{customerId}/{projectId}/workspaces/{workspaceId}/credentials/{credentialId}/details/{credentialsName}`.
+
+
+## Publishing to the Adobe Experience Platform
+
+To publish your Avo tracking plan to the Adobe Experience Platform, navigate to the AEP integration view in Avo and click the "Publish" button. Enable Auto Publishing to trigger an automatic publish whenever a branch has been merged.
```

Comment:
Two things here: 
1/ I think "navigate to the AEP integration view in Avo and click the "Publish" button" could be clearer as something like "open the integration details of the AEP integration that you just created"

2/ Can we have a screenshot here that visualizes this? That it's clear you are in publishing in the sidebar and that you have clicked the integration to open it